components/sub-select.js

```diff
@@ -94,8 +94,13 @@ export default function SubSelect ({ prependSubs, sub, onChange, size, appendSub
           }
         } else {
           // we're currently on the home sub
-          // are we in a sub aware route?
-          if (router.pathname.startsWith('/~')) {
+          // if we're in top/cowboys and a territory is selected,
+          // go to ~sub/top/posts/day
+          if (router.pathname.endsWith('top/cowboys')) {
```

Comment:
I was only wondering if this would make this a little more reliable but I am about to leave for the day so I am just leaving this suggestion that we can probably ignore:

```suggestion
          if (router.pathname.endsWith('/top/cowboys')) {
```